Snowplow/Internal/Subject/SPPlatformContext.m

```diff
@@ -102,6 +100,14 @@ - (void) setMobileDict {
 - (void) setEphemeralMobileDict {
     self.lastUpdatedEphemeralMobileDict = [[NSDate date] timeIntervalSince1970];
     
+    NSDictionary *currentDict = [self.platformDict getAsDictionary];
+    if ([currentDict valueForKey:kSPMobileAppleIdfa] == nil) {
```

Comment:
nitpicky:
You could also write `if (![currendDict valueForKey:kSPMobileAppleIdfa]) {` as the nil is directly considered false by objc.
Also, I always forget the difference between `objectForKey` and `valueForKey`. I'm sure `valueForKey` is fine but I can't remember the difference.
Just a thought btw. No need to change anything. It's already 👍 in this way.